subsys/bluetooth/host/att.c

```diff
@@ -105,9 +105,14 @@ struct bt_att {
 	/* Contains bt_att_chan instance(s) */
 	sys_slist_t		chans;
 #if defined(CONFIG_BT_EATT)
-	struct k_work_delayable connection_work;
-	uint16_t previous_ecred_connection_result;
-	uint8_t eatt_chans_to_connect;
+	struct {
+		struct k_work_delayable connection_work;
+		uint8_t eatt_chans_to_connect;
+
+		uint16_t prev_eatt_conn_rsp_result;
+		uint16_t prev_eatt_conn_req_result;
+		uint8_t prev_eatt_conn_req_missing_chans;
+	} conn_context;
```

Comment:
Why not just call this `eatt` instead of `conn_context`? You could then also remove any mentions of "eatt" in the struct member names, since it's already implied.